### PR TITLE
You might need to reject this...

### PR DESCRIPTION
**Pull request by whyderrick**
Handful of minor tweaks that will hopefully be the finale. Possible that I did something wrong in trying to handle this through the command line. 

I couldn't run the ruby script correctly (got this error: `fatal: /Users/derrickcarr/Documents/Alight: '/Users/derrickcarr/Documents/Alight' is outside repository`) so I'll need  you to run it so I can take a look  at the new additions to the FrontMatter.html doc. There's also asterisks in the `Megan Falley - Watching my Mother.html` that seem to be transposed from `&ast;` to `&amp;&ast;` through somewhere in the script process. 

Talk to ya soon!

### CODE REVIEW

File: alight/content/Amaris Diaz - DEAR PRIVILEGE.html [hunk 1]
```diff
@@ -18,7 +18,7 @@ <h3 class="Section poem-title" id="amarisdiaz-dearprivilege">DEAR PRIVILEGE</h3>
 	</div>
 	<div class="PoemStanza">
 		<p class="PoemLine1">Dear White Student Union,</p>
-		<p class="PoemLine">I am sorry I have taken so much of your privilege</p>
+		<p class="PoemLine">I am sporry I have taken so much of your privilege</p>
```
Comment (mdayaram) [inline alight/content/Amaris Diaz - DEAR PRIVILEGE.html:21]: Is this really supposed to be `sporry` instead of `sorry`?

Comment (whyderrick) [inline alight/content/Amaris Diaz - DEAR PRIVILEGE.html:21]: Nope. It's definitely not supposed to be that. Thanks for reviewing.

**Comment by whyderrick** [comment]
I have no idea how pull requests work, but I added the following css block to style.css just now in commit `8f9f715`

```
#front-matter h1, #front-matter h2 {
    background-color: inherit;
}
```

**Comment by mdayaram** [comment]
Closing this pull request since we got the changes in from another PR.

**Pull request closed without merging**